source/_integrations/modbus.markdown

```diff
@@ -827,6 +827,11 @@ sensors:
       description: "If `data_type` is custom specified a double-quoted Python struct is expected here, to format the string to unpack the value. See Python documentation for details. Example: `>i`."
       required: false
       type: string
+    register_size:
+      description: Register size, number of bytes of 1 count. For non-standard registers only!
```

Comment:
This integration (as stated in the top) implements the standard modbus protocol, as defined by the modbus organization.

Changing the register_size is not in the standard protocol nor documented by the modbus organization, so there must a be a pointer to a standard where register_size is documented with it´s limits and/or other definitions. The standard must define how it relates to the modbus standard which this integration follows.

If such a standard exist, and it defines the relationship to the modbus standard, it can be considered to change the objective of this integration. Without a standard to document the why/how it opens the modbus integration to implement just about anything.